### PR TITLE
Item Fixes (Submaster? Rohoxna Cliff?) LINES 9571 to 9708 ONLY (Ignore the auto's above this)

### PR DESCRIPTION
**Pull request by Sourpusss**
**_Lines 9571 - 9708**_

**_I only intended to work on the above lines for this edit.  Please ignore the weird auto changes that were added and then removed in the updated edit.  The first commit contains the actual edit, of which I have marked the start of the actual edited portion via a comment.**_

Added some punctuation and made some wording more fluid.

Lines 9577+ mention the Hoplite / Pyro / Etc Submasters - just to clarify, these are specifically referring to the Submasters, and not the Masters?  I know there are multiple illustrated NPC's of certain classes, so I'm not sure if this an actual designation or just some weird translation thing.  Later on the "Rogue Master's Pouch" (line 9587) is mentioned.  Is this the Rogue Master's Pouch, or the Submaster's Pouch?  Also the Pardoner Master is mentioned.

Line 9625/6 says "Scout Master's Special Medicine."  In this context, I'm assuming it's some chemical used to reveal some hidden words on paper, so I'm going to call it a reagent, as Medicine seems like it's something that the Scout Master ingests.  I also changed "drug" to chemical for this same reason.

Line 9652:  What is Rohoxna Cliff?  Roxana?  I don't know - I looked at the Korean line and couldn't figure out how this was part of the translation (I couldn't find a named cliff).  There were some other names mentioned (not part of the english translation).  I feel like just "the cliff" would suffice unless a proper name can / was already given.

Line 9662: Should "Demon Warrant" be something like "Demon Ward" or "Demon Writ?"  Warrant is kind of an odd word to describe something sealing away something (which I assume is what this does).  I'll edit the first sentence after this is cleared up.

9679:  The first sentence was redundant and awkward. 

9688-91:  Just to make sure my wording works - these items are part of a delivery quest from Rose to be delivered to Laswi?  If they're handed across npc's infront of the player as part of some cutscene / quest interaction, this wording doesn't really work, so please let me know.

It's been a while since I've worked on any edits, and from what I can tell, the whole $ thing isn't around anymore?  Is it strictly forbidden now, or optional?  Also, my bad for going over 150.

### CODE REVIEW

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -689,162 +689,162 @@ ITEM_20150317_000688	Breach Pike
...
+ITEM_20150317_000822	Used as a support weapon but very capable of defending or deflecting enemy attacks. 
+ITEM_20150317_000823	A short dagger with double edge. It's an evolved form of the tool knife used by the Highlanders with an elongated blade. 
+ITEM_20150317_000824	A dagger made to be easily used left-handed. Designed with the purpose of blocking an opponent's sword by using the Guard skill. The grip is mirrored so that it fits well to your left hand. 
 ITEM_20150317_000825	Chura 
-ITEM_20150317_000826	Single-edged, sharply pointed dagger. A specialized design for stabbing.
+ITEM_20150317_000826	Single-edged, sharply pointed dagger. Specialized design for stabbing. 
 ITEM_20150317_000827	Pescas 
-ITEM_20150317_000828	Single-edge dagger with extremely sharp, long thin tip. It has an 'S' shape body with a heavy, delicately decorated hilt.
+ITEM_20150317_000828	Single-edge dagger with extremely sharp, long thin tip. It has a 'S' shape body with a heavy, delicately decorated hilt.
```
Comment (WatchGintama) [inline EnglishTranslation-master/ITEM.tsv]: > an 'S' 

is correct

File: EnglishTranslation-master/ITEM.tsv [hunk 2]
```diff
@@ -9170,7 +9170,7 @@ ITEM_20151224_009169	Acquiring equipment worn by demons is an extremely difficul
 ITEM_20151224_009170	Blue Temple Slave Hood
 ITEM_20151224_009171	Stealthy Temple Slaves used their hoods to conceal themselves. {nl}Obtained from Blue Temple Slaves.
 ITEM_20151224_009172	Green Tiny Tail
-ITEM_20151224_009173	Rumor has it that a few eccentric individuals collect this item. {nl}Obtained from Green Tinys.
+ITEM_20151224_009173	Rumour has it that a few eccentric individuals collect this item. {nl}Obtained from Green Tinys.
```
Comment (WatchGintama) [inline EnglishTranslation-master/ITEM.tsv]: > Rumour

should be rumor

File: EnglishTranslation-master/ITEM.tsv (hunk 1 above)
Comment (Sourpusss) [inline EnglishTranslation-master/ITEM.tsv]: Bluh, notepad+ or pasting it back or SOMETHING did some autocorrecting.  Thanks for the catching this and the other thing.  I didn't even read the lines, let alone edit them, outside of the ones listed in my main comment.  

Edit: Oh wow, it added a buncha ". " too.

File: EnglishTranslation-master/ITEM.tsv [hunk 3]
```diff
@@ -9568,31 +9568,31 @@ ITEM_20151224_009567	A strange letter written by a mysterious mage to an unknown
 ITEM_20151224_009568	Mage's Second Letter
 ITEM_20151224_009569	A letter sent to someone known as Delmore Rephaim by a mysterious mage. Readable.
 ITEM_20151224_009570	Officer's Journal Part 1
-ITEM_20151224_009571	The first part written by an officer of the Delmore Castle regarding his work. Readable
```
Comment (Sourpusss) [inline EnglishTranslation-master/ITEM.tsv:9571]: **This is where the edit actually starts**

File: EnglishTranslation-master/ITEM.tsv [hunk 4]
```diff
@@ -9601,111 +9601,111 @@ ITEM_20151224_009600	Spell Detector Orb
...
 ITEM_20151224_009697	A counteragent given by the Cleric Submaster. It neutralizes sinister energy.
 ITEM_20151224_009698	Dadan Jungle Experiment Journal, Chapter 1
-ITEM_20151224_009699	An experimental jounral written in a human language. Right-click to use.
+ITEM_20151224_009699	An experimental journal written by humans. Right-click to use.
 ITEM_20151224_009700	Dadan Jungle Experiment Journal, Chapter 2
 ITEM_20151224_009701	Dadan Jungle Experiment Journal, Chapter 3
 ITEM_20151224_009702	Dadan Jungle Experiment Journal, Chapter 4
 ITEM_20151224_009703	Green Apparition Essence
-ITEM_20151224_009704	An essence to prove themselves as a demon to open the special reading room.
+ITEM_20151224_009704	An essence used to prove themselves a demon, in order to open the special reading room.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: special reading room -> Special Reading Room (it's a map name)

File: EnglishTranslation-master/ITEM.tsv [hunk 5]
```diff
@@ -9601,111 +9601,111 @@ ITEM_20151224_009600	Spell Detector Orb
...
 ITEM_20151224_009645	Priest Irma's Journal
 ITEM_20151224_009646	Collected pages of Priest Irma's Journal.
-ITEM_20151224_009647	A note found in a bundle of grass with smoke coming out. Half of the content is burnt. 
-ITEM_20151224_009648	A drug created coincidentally from a failure by Germeja. The monsters will fight against each other if used on them.
+ITEM_20151224_009647	A note found in a smoking bundle of grass. Half of the content is burnt. 
+ITEM_20151224_009648	A drug created coincidentally from Germeja's failure. The monsters will fight against each other if it's used on them.
 ITEM_20151224_009649	A note left by Priest Gelija. This might be the decisive clue since Gelija is so thorough.
 ITEM_20151224_009650	A whistle that can call Jolly to your location. Blow the whistle if Jolly is too far away.
 ITEM_20151224_009651	Leftover Box
 ITEM_20151224_009652	The blood of monsters found in Rohonxa Cliff. Is it contaminated due to the demons?
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Rohonxa Cliff > Rohonsa Cliff

**Comment by Sunaries** [comment]
> Lines 9577+ mention the Hoplite / Pyro / Etc Submasters - just to clarify, these are specifically referring to the Submasters, and not the Masters? I know there are multiple illustrated NPC's of certain classes, so I'm not sure if this an actual designation or just some weird translation thing. Later on the "Rogue Master's Pouch" (line 9587) is mentioned. Is this the Rogue Master's Pouch, or the Submaster's Pouch? Also the Pardoner Master is mentioned.

A `Master` and `Submaster` are two different NPCs. The pouch is for the `Rogue Master`.

> Line 9625/6 says "Scout Master's Special Medicine." In this context, I'm assuming it's some chemical used to reveal some hidden words on paper, so I'm going to call it a reagent, as Medicine seems like it's something that the Scout Master ingests. I also changed "drug" to chemical for this same reason.

https://tos.neet.tv/items/661079 - This is how it looks.

> Line 9652: What is Rohoxna Cliff? Roxana? I don't know - I looked at the Korean line and couldn't figure out how this was part of the translation (I couldn't find a named cliff). There were some other names mentioned (not part of the english translation). I feel like just "the cliff" would suffice unless a proper name can / was already given.

I commented on the line. Should be: `Rohonsa Cliff` 

`Line 9662: Should "Demon Warrant" be something like "Demon Ward" or "Demon Writ?" Warrant is kind of an odd word to describe something sealing away something (which I assume is what this does). I'll edit the first sentence after this is cleared up.`

I don't know if `Orders` is synonymous with `Warrant`? From reading it, it's `Demon Orders`.

**Keep in mind that these items, especially the Scout Master's item are also displayed in other files, such as quests and ETC. So you need to change them there as well to ensure consistency.**

> It's been a while since I've worked on any edits, and from what I can tell, the whole $ thing isn't around anymore? Is it strictly forbidden now, or optional? Also, my bad for going over 150.

IMC removed the $ marks. We don't do it anymore, even though there was no clarification why. 
For iTOS, the $ needed to be removed anyway. I believe it is not necessary _anymore_.
Also, we don't mind if you go over 150, as long as they are in separate commits so that everyone can see all the changes. :smile:

File: EnglishTranslation-master/ITEM.tsv [hunk 6]
```diff
@@ -9601,111 +9601,111 @@ ITEM_20151224_009600	Spell Detector Orb
...
 ITEM_20151224_009653	Chaser's Package
 ITEM_20151224_009654	A bag filled with medicine and bandages.
-ITEM_20151224_009655	A red crystal once held by demons. Once could very well be threatened because of it.
-ITEM_20151224_009656	A wood piece with a corruptive energy held by the demons. Can it drive out the curse...
-ITEM_20151224_009657	A mysterious pack held by every demons. It might be related to the curse.
-ITEM_20151224_009658	A mysterious pack held by every demons. Try using it figure out the effect.
+ITEM_20151224_009655	A red crystal once held by demons. It wouldn't be out of the question to feel threatened by it.
+ITEM_20151224_009656	A wood piece held by demons emitting a corruptive energy. Can it drive out the curse?
+ITEM_20151224_009657	A mysterious pack held by every demon. It might be related to the curse.
+ITEM_20151224_009658	A mysterious pack held by every demon. Try using it figure what it does. Don't try this at home!
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Try using it figure what it does -> Try using it to figure out what it does.
"Don't try this at home!" is contradicting. The description says to open it, but then followed by a sentence not to.

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ITEM.tsv [hunk 7]
```diff
@@ -9601,111 +9601,111 @@ ITEM_20151224_009600	Spell Detector Orb
 ITEM_20151224_009601	An orb that can detect the flow of spells. Who could have made such a thing?
 ITEM_20151224_009602	A flask for collecting spiritual energy. It feels a bit creepy.
 ITEM_20151224_009603	Delmore Outskirts Gun Powder
-ITEM_20151224_009604	Gun powder made in the outskirts of Delmore requested by the Sapper Submaster. Where could the civilians have gone?
+ITEM_20151224_009604	Gun powder made in the outskirts of Delmore, requested by the Sapper Submaster. Where could the civilians have gone?
 ITEM_20151224_009605	Wugushi Poison Bottle
-ITEM_20151224_009606	A bottle that can carry the poison extracted from a corpse. Uhm... What to say...
+ITEM_20151224_009606	A bottle that can carry poison extracted from corpses. Uhm...What to say...
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Uhm...What to say... -> Uhm... What to say...

Don't remove the space.

**Pull request closed without merging**